azure_functions_worker/bindings/meta.py

```diff
@@ -12,21 +12,23 @@
 PB_TYPE = 'rpc_data'
 PB_TYPE_DATA = 'data'
 PB_TYPE_RPC_SHARED_MEMORY = 'rpc_shared_memory'
+BINDING_REGISTRY = None
 
 
-def get_binding_registry():
+def load_binding_registry() -> None:
     func = sys.modules.get('azure.functions')
 
     # If fails to acquire customer's BYO azure-functions, load the builtin
     if func is None:
         import azure.functions as func
 
-    return func.get_binding_registry()
+    global BINDING_REGISTRY
+    BINDING_REGISTRY = func.get_binding_registry()
```

Comment:
We need to test this specifically - to ensure that there is always one call.

Reply:
The unit test change which I made make sure that this is called atleast once. Also the consumption e2e test tests this call.